sdk/imagebuilder/arm-imagebuilder/src/operations/virtualMachineImageTemplates.ts

```diff
@@ -91,6 +98,31 @@ export class VirtualMachineImageTemplatesImpl
     }
   }
 
+  /**
```

Comment:
The following lines are moved for viewing the code easily. They are not actual code changes